department-of-veterans-affairs/caseflow
# Ki/APPEALS-35482 - Correspondence Cases - Unassigned Tab

Resolves [APPEALS-35482](https://jira.devops.va.gov/browse/APPEALS-35482)

# Description
Updates for the _Unassigned_ tab on the Correspondence/Team cases page.  It includes sorting functionality and the updated (visual-only) components to match the wireframes.
- Updated front-end to match wireframes
  - User dropdown, "Assign", and "Auto assign correspondence" buttons, horizontal rule (functionality in future ticket)
- Verified changes that all sorting and filtering work on all columns in Unassigned tab on team page
- Updated RSpecs to test Unassigned Tab

## Acceptance Criteria

## Testing Plan

1. Navigate to Correspondence/Team page
2. Verify population of front-end changes only on Unassigned tab (also changes for Assigned tab)
3. Verify filtering and sort work for the 4 filterable columns
    1. Veteran Details
    2. VA DOR
    3. Days Waiting
    4. Notes

# Frontend
## User Facing Changes
![Screenshot 2024-02-21 at 4 12 01 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/2c003ca6-607b-485f-b84a-b64bc3077681)
![Screenshot 2024-02-21 at 4 12 12 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/22d3bcba-0dc5-4fba-a1ef-1518fc5acc0f)


## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
